implementations/graphene/server.py

```diff
@@ -53,60 +142,60 @@ def __resolve_reference(self, info, **kwargs):
             return Product(**get_product_by_sku_and_variation(self.sku, self.variation))
 
 
+# @key(fields="sku package")
+class DeprecatedProduct(ObjectType):
+    sku = String(required=True)
+    package = String(required=True)
+    reason = String()
+    createdBy = Field(User)
+
+
 class Query(ObjectType):
     product = Field(Product, id=ID(required=True))
+    deprecatedProduct = Field(DeprecatedProduct, sku=String(required=True), package=String(required=True),
```

Comment:
```suggestion
    deprecated_product = Field(DeprecatedProduct, sku=String(required=True), package=String(required=True),
```

Reply:
oh my bad thought I fixed all of that! thanks!